data/pigui/modules/new-game-window/game-param.lua

```diff
@@ -0,0 +1,40 @@
+-- Copyright © 2008-2023 Pioneer Developers. See AUTHORS.txt for details
+-- Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+local utils = require 'utils'
+local GameParam = utils.class("pigui.modules.new-game-window.game-param")
```

Comment:
Recommend annotating this with `---@class NewGameWindow.GameParam` or similar for autocompletion.

Reply:
Annotated.